src/redux/tasks/types.ts

```diff
@@ -1,6 +1,9 @@
+import { ITasksList } from '../tasksLists/types'
+
 export type ITask = {
     id: number,
-    title: string, 
-    description: string
-    isDone: boolean
+    name: string, 
+    pos: number,
+    state: boolean,
```

Comment:
state ? Why not `isChecked` ? Seems more meaningful to me

Reply:
This is the name defined in Trello-routes, so I left state.